### PR TITLE
add python lambda app to calculate total kin in circulation

### PR DESCRIPTION
**Pull request by oryband**

### CODE REVIEW

File: apps/kin-in-circulation/app.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    async with ClientSession(
+            raise_for_status=True,
+            timeout=ClientTimeout(total=HORIZON_REQUEST_TIMEOUT),
+    ) as client:
+
+        balances = await get_balances(client)
+
+    tokens_in_circulation = calculate_kin_in_cicrulation(TOTAL_KIN_AMOUNT, balances)
+
+    return float(tokens_in_circulation)
```
Comment (Ronserruya) [inline apps/kin-in-circulation/app.py]: Maybe return string? Just so you don't have floating point issues.

Comment (Ronserruya) [inline apps/kin-in-circulation/app.py]: Didnt you need to return some json with response code and body?

Comment (oryband) [reply apps/kin-in-circulation/app.py]: Python Decimal plays well with `float()`. There won't be non-human weird rounding surprises here

Comment (oryband) [reply apps/kin-in-circulation/app.py]: I didn't know at the time but we only need to return this for now.

**Comment by oryband** [comment]
i force removed this merged branch from master

**Pull request merged**